README.md

```diff
@@ -0,0 +1,36 @@
+# Exploratory Data Analysis and Time Series Modeling for Rossmann Store Sales
+
+## Project Description
+This project focuses on exploratory data analysis and time series modeling of sales data from 1,115 Rossmann stores. The data used in the project is sourced from Kaggle. The goal of the project is to understand sales dynamics, detect trends and seasonality, and build effective models for predicting sales in individual stores.
+
+## Data
+The data used in the project was provided by Kaggle as part of the "Rossmann Store Sales" competition. The dataset contains information about sales in 1,115 Rossmann stores, along with data regarding promotions, competition, and store information.
+
+## Required Libraries
+The project utilizes several popular Python libraries, including:
+- pandas
+- numpy
+- matplotlib
+- seaborn
+- scikit-learn
+- statsmodels
+
```

Comment:
na ten moment na pewno jeszcze os

Reply:
os is part of the standard library, so if you have Python, you have os already.